scientific/src/commonMain/kotlin/unit/DimensionlessUnit.kt

```diff
@@ -0,0 +1,34 @@
+/*
+ Copyright 2022 Splendo Consulting B.V. The Netherlands
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+ */
+
+package com.splendo.kaluga.scientific.unit
+
+import com.splendo.kaluga.base.utils.Decimal
+import com.splendo.kaluga.scientific.PhysicalQuantity
+import kotlinx.serialization.Serializable
+
+@Serializable
+sealed class Dimensionless : ScientificUnit<PhysicalQuantity.Dimensionless>,
+    MetricAndImperialScientificUnit<PhysicalQuantity.Dimensionless>
+
+object Percent : Dimensionless(), MetricBaseUnit<MeasurementSystem.MetricAndImperial, PhysicalQuantity.Dimensionless> {
```

Comment:
I'd name it Percentage

Reply:
+1 for Percentage

Reply:
According to Technical writing guides from both MIT and Penn use of "Percentage" would be incorrect for this.

> “Percentage” is used to refer to a general relationship rather than a specific measure:
> A large percentage of the people voted, but only 20 percent of the votes counted.

source: https://www.e-education.psu.edu/styleforstudents/c3_p43.html

> The word percentage refers to a general result or portion.
> The percentage of system failures was below the limit set for the prototype.

source: http://www.mit.edu/course/21/21.guide/percent.htm